projects/ccf-rui/src/app/core/store/model/model.state.ts

```diff
@@ -270,6 +273,7 @@ export class ModelState extends NgxsImmutableDataRepository<ModelStateModel> {
    */
   @DataAction()
   setPosition(position: XYZTriplet): void {
+    this.ga.event('position', 'position_set', `${position.x} ${position.y} ${position.z}`);
```

Comment:
Include the organ name in this event.
```typescript
    this.ga.event('placement', `${this.snapshot.organ.name}_placement`, `${position.x} ${position.y} ${position.z}`);
```

Reply:
You may need to round the positions btw. Yeah, actually round up to 1 decimal place and underscore separate them instead of space.